crypto/sm2/sm2_sign.c

```diff
@@ -230,7 +324,7 @@ ECDSA_SIG *SM2_do_sign(const EC_KEY *key,
     BIGNUM *e = NULL;
     ECDSA_SIG *sig = NULL;
 
-    e = compute_msg_hash(digest, key, user_id, msg, msg_len);
+    e = SM2_compute_msg_hash(digest, key, user_id, msg, msg_len);
     if (e == NULL)
```

Comment:
No error log because we already did it in the other function? It might be nice to leave a comment here (and elsewhere) where that is the case.